pkg/block/index.go

```diff
@@ -665,8 +670,22 @@ func rewrite(
 		return labels.Compare(series[i].lset, series[j].lset) < 0
 	})
 
+	lastSet := labels.Labels{}
 	// Build a new TSDB block.
 	for _, s := range series {
+		// The TSDB library will throw an error if we add a series with
```

Comment:
Can we remove double spaces in this comment?

Reply:
Old habits die hard...fixed.

Reply:
Can I ask where is this habit from? Sorry for my ignorance - super curious (:

Reply:
Lol...  Search google for "double space after period" you'll learn more than you bargained for.  Like, my age.  :-)

Reply:
TIL never heard of this either. But actually the rule could apply here since code formatting is using monospaced fonts mostly :)